examples/grpc.core/Examples.GrpcCore.AspNetCore/Examples.GrpcCore.AspNetCore.csproj

```diff
@@ -1,13 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry.Exporter.Console" Version="1.0.1" />
-    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc2" />
-    <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.0.0-rc2" />
+    <PackageReference Include="OpenTelemetry.Exporter.Console" Version="$(OpenTelemetryCoreLatestVersion)" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc9.4" />
```

Comment:
Great to see updated examples.
I have some doubts about using configuration variables `OpenTelemetryCoreLatestVersion`.
It is additional layer to understand by endusers.

Reply:
OpenTelemetry main repo does not have the package versions explicitly mentioned in the .csproj of example projects either.  Ideally, these things should be mentioned in the README.md for these projects.

I don't have a strong opinion here. I can change these back to the specific versions. Let me know.

Reply:
It is easier to maintain current version. Lets keep it as it is.